pkg/providers/vault/examples/pv-keyvault-csi.yaml

```
@@ -0,0 +1,25 @@
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: pv-keyvault
+spec:
+  capacity:
+    storage: 1Gi
+  accessModes:
+    - ReadOnlyMany
+  persistentVolumeReclaimPolicy: Retain
+  csi:
+    driver: keyvault.csi.k8s.com
```

```suggestion
    driver: secrets-store.csi.k8s.com
```

https://github.com/deislabs/secrets-store-csi-driver/blob/master/deploy/example/pv-secrets-store-csi.yaml#L12